v20.1/cockroachcloud-quickstart.md

```diff
@@ -41,33 +41,32 @@ Once your cluster is created, you will be redirected to the **Cluster Overview**
 
 1. In the left navigation bar, click **SQL Users**.
 2. Click **Add User**. The **Add User** modal displays.
-3. For **Username**, enter `trial_user`.
-4. For **Password**, enter `trial_password`.
+3. Enter a **Username** and **Password**.
 4. Click **Save**.
 
 ### Step 2. Authorize your network
 
 1. In the left navigation bar, click **Networking**.
 2. Click **Add Network**. The **Add Network** modal displays.
-3. From the **Network** dropdown, select **Public (Insecure)** to allow all networks.
-
-     {{site.data.alerts.callout_info}}
-     Use this with caution as your cluster will be vulnerable to denial-of-service and brute force password attacks.
-     {{site.data.alerts.end}}
-
+3. From the **Network** dropdown, select **Current Network** to auto-populate your local machine's IP address.
```

Comment:
Tried to retain the "local quickstart" experience here by asking them to authorize only their local network and not explain the three network options (public, current, custom) in detail. Hope that's okay. LMK if it isn't!